villains-only-sts/src/main/java/com/villains/controller/PictureController.java

```diff
@@ -43,13 +43,13 @@
 	
 	@PostMapping("addPicture.app")
 	public @ResponseBody ResponseEntity<Message> addPicture(@RequestBody Picture picture){
+		System.out.println(picture.getPostId().getPostId());
 		pictureService.addPicture(picture);
 		return new ResponseEntity<>(new Message("Picture Successfully Uploaded!"), HttpStatus.OK);
 	}
 	
 	@PostMapping("editPicture.app")
 	public @ResponseBody ResponseEntity<Message> editPicture(@RequestBody Picture picture){
-		pictureService.editPicture(picture);
```

Comment:
Why is this taken out? Were there changes in the db on how we would handle pictures, post, likes, etc. that need this line taken out?